straxen/plugins/event_processing.py

```diff
@@ -128,11 +128,23 @@ def compute(self, peaks, start, end):
         name='force_main_before_alt', default=False, infer_type=False,
         help="Make the alternate S1 (and likewise S2) the main S1 if "
              "occurs before the main S1."),
+    strax.Option(
+        name='force_alt_s2_in_max_drift_time', default=True, infer_type=False,
+        help="Make sure alt_s2 is in max drift time starting from main S1"),
     strax.Option(
         name='event_s1_min_coincidence',
         default=2, infer_type=False,
         help="Event level S1 min coincidence. Should be >= s1_min_coincidence "
              "in the peaklet classification"),
+    strax.Option(
+        name='electron_drift_velocity', infer_type=False,
+        default=("electron_drift_velocity", "ONLINE", True),
+        help='Vertical electron drift velocity in cm/ns (1e4 m/ms)',),
```

Comment:
could we replace this with an URL config? We are now changing the hash anyway.

Reply:
Sure! I'm wondering if you can provide an example of URL config? Here I'm just copying from class Events' config.

Reply:
you can set it as a class attribute (see e.g. https://github.com/XENONnT/straxen/blob/92877a183f20aaf425625a5f00af30ac1f0fec31/straxen/plugins/event_processing.py#L577) 
```python
    electron_drift_velocity = straxen.URLConfig(
        default='cmt://electron_drift_velocity?version=ONLINE&run_id=plugin.run_id',
        help='Vertical electron drift velocity in cm/ns (1e4 m/ms)')
``` 
you can remove this from the `setup()` then.

Let me know if you want me to push this commit / fix it when straxen starts complaining (I think `Events` should be similarly updated).